### ISSUE TITLE
keys.gnupg.net is obsolete

### ISSUE THREAD

**Issue by seetamraju** [aws/copilot-cli#3944]
Hi team,
# ((PROBLEM #1 ))
We have outdated instructions at https://aws.github.io/copilot-cli/docs/getting-started/verify/
Specifically, I'm talking about the use of `keys.gnupg.net` as the **KEYSERVER**.
That FQDN is now OBSOLETE.
**DNS resolution of that domain fails!!!!**
Please reference: https://dev.gnupg.org/T5527
## ------ POTENTIAL SOLUTION ------
Remove keyserver from command.
SIMPLIFY the gpg command as:
```
# gpg  --recv BCE9D9A42D51784F
gpg: key BCE9D9A42D51784F: new key but contains no user ID - skipped
gpg: Total number processed: 1
gpg:           w/o user IDs: 1
```
## ----- Also, pls note: ------
Using `GPG Keychain App v1.10` on MacOS, the keylookup of `BCE9D9A42D51784F` on `hkps://keys.openpgp.org` ALSO FAILS.
Please upload key to that KEYSERVER (`hkps://keys.openpgp.org`)

# ((PROBLEM #2 ))
Instruction re: validing the signature also fails.
```
# $ gpg --verify copilot.asc /usr/local/bin/copilot
gpg: Signature made Tue Aug 16 19:13:25 2022 EDT
gpg:                using RSA key DE3CBD61ADAF8B8E
gpg: Can't check signature: No public key
```
# ---- I do NOT have a solution ----
VERY LIKELY because Problem #1 is still outstanding issue for me.

**Comment by seetamraju** [comment]
Update.
# Found Solution
switch KEYSERVER to `keyserver.ubuntu.com`

```
$ gpg --keyserver keyserver.ubuntu.com --recv BCE9D9A42D51784F
gpg: key BCE9D9A42D51784F: public key "Amazon ECS <ecs-security@amazon.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1


$ gpg --verify copilot.asc /usr/local/bin/copilot
gpg: Signature made Tue Aug 16 19:13:25 2022 EDT
gpg:                using RSA key DE3CBD61ADAF8B8E
gpg: Good signature from "Amazon ECS <ecs-security@amazon.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: F34C 3DDA E729 26B0 79BE  AEC6 BCE9 D9A4 2D51 784F
     Subkey fingerprint: EB3D F841 E2C9 212A 2BD4  2232 DE3C BD61 ADAF 8B8E
```

# NEW PROBLEM !!!

Please note the warning above = *This key is not certified with a trusted signature*

**Comment by Lou1415926** [comment]
Hello @seetamraju !

Thank you very much for pointing this out! I can confirm that public key can be retrieved from `keyserver.ubuntu.com`. If you have the bandwidth, feel free to open a PR to update the keyserver from `hkp://keys.gnupg.net` to `keyserver.ubuntu.com`. Otherwise, I can do so at the end of the week.

> Please note the warning above = This key is not certified with a trusted signature

The warning is expected and is not problematic. It occurs because there is not a chain of trust between your personal PGP key (if you have one) and the Amazon ECS PGP key. For more information, see [Web of trust](https://en.wikipedia.org/wiki/Web_of_trust).

**Comment by seetamraju** [comment]
Hi @Lou1415926 ,  Will do exactly that - PR.  Pls give me 2 days.

**Comment by efekarakus** [comment]
This fix is now released in [v1.22](https://github.com/aws/copilot-cli/releases/tag/v1.22.0) 🚀 !

For the blog post: https://aws.github.io/copilot-cli/blogs/release-v122/
Release notes: https://github.com/aws/copilot-cli/releases/tag/v1.22.0
